README.md

```diff
@@ -68,6 +70,41 @@ export default function App() {
 }
 ```
 
+#### Option 2: Declaring a sketch using abstracted setup and draw functions
+
+```js
+import React, { useState, useEffect } from "react";
+import { ReactP5Wrapper } from "react-p5-wrapper";
+
+function setup(p5) {
+  return () => {
+    p5.createCanvas(600, 400, p5.WEBGL);
+  };
+}
+
+function draw(p5) {
+  return () => {
+    p5.background(250);
+    p5.normalMaterial();
+    p5.push();
+    p5.rotateZ(p5.frameCount * 0.01);
+    p5.rotateX(p5.frameCount * 0.01);
+    p5.rotateY(p5.frameCount * 0.01);
+    p5.plane(100);
+    p5.pop();
+  };
+}
+
+function sketch(p5) {
+  p5.setup = setup(p5);
+  p5.draw = draw(p5);
+}
+
+export default function App() {
```

Comment:
Maybe we want to also update the readme to stop exporting as `default` in the examples and instead prefer names exports since we also do that in the source code for the wrapper component. It's not strictly related to the PR but what do you think of this?

Reply:
Sounds reasonable - updated.